### PR TITLE
[passkit] Update for beta 1-2 for iOS and watchOS

### PR DESCRIPTION
**Pull request by spouliot**
Note: several ArgumentSemantic were updated in the header files

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by xamarin-release-manager** [comment]
Build failure

File: src/passkit.cs [hunk 1]
```diff
@@ -350,36 +366,46 @@ interface PKPaymentRequest {
...
+		[Export ("paymentData", ArgumentSemantic.Copy)]
 		NSData PaymentData { get; }
 
 		[iOS (9,0)]
-		[Export ("paymentMethod")]
+		[Export ("paymentMethod", ArgumentSemantic.Strong)]
 		PKPaymentMethod PaymentMethod { get; }		
 	}
 
+#if !WATCH
```
Comment (VincentDondain) [inline src/passkit.cs]: Why not using `[NoWatch]` on all the types here?

Comment (spouliot) [inline src/passkit.cs]: Minimizing changes in branches makes merging easier: less conflicts, less time required, less mistakes... less headaches.

**Pull request merged**